cms/envs/devstack.py

```diff
@@ -189,6 +193,17 @@ def should_show_debug_toolbar(request):
     'http://localhost:18150/logout/',  # credentials
 ]
 
+###################### Cross-domain requests ######################
+FEATURES['ENABLE_CORS_HEADERS'] = True
+CORS_ALLOW_CREDENTIALS = True
+CORS_ORIGIN_WHITELIST = (
+    'http://localhost:3001',
+)
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Nit: afaik using CORS_ORIGIN_ALLOW_ALL, makes the CORS_ORIGIN_WHITELIST redundant. Do we need to set both of them?

Reply:
Looking at `lms/envs/devstack.py`, `CORS_ORIGIN_ALLOW_ALL = True`, should be sufficient.

Reply:
Agreed.